toolchain/driver/driver.h

```diff
@@ -48,6 +48,13 @@ class Driver {
   // error stream (stderr by default).
   auto RunCommand(llvm::ArrayRef<llvm::StringRef> args) -> RunResult;
 
+  // Finds the source files that define the prelude and returns a list of their
+  // filenames. On error, writes a message to `error_stream` and returns an
+  // empty list.
+  static auto FindPreludeFiles(llvm::StringRef data_dir,
```

Comment:
This is taking in members of Driver, so why a static function? I'd expect everywhere that uses it is creating an instance of Driver, and it could be called before RunCommand.

Reply:
It's used to construct the `llvm::vfs::FileSystem` that is passed to the driver, and doesn't respect that file system, so I felt it was clearer to make it static rather than to make it non-static and have `file_test` mutate the VFS after passing it into the driver.